content/en/hosting/costs/_index.md

```diff
@@ -0,0 +1,95 @@
+
+When analyzing the hosting total cost of ownership (TCO), only production instances were looked at.
+
+It should be noted that a development environment can be no cost (or very low cost).  Assuming a developer already has a laptop, this is all that is needed to host a development instance. 
+
+Running an easy to set up Docker based development environment (instead of Kubernetes) on a laptop is an acceptable practice: there is no expectation of high uptime for development instances.
+
+## Expenses behind the per user cost
+
+To be clear, this document is very much a hosting TCO and not a generic TCO document.  As such, below is a list of what's included in the hosting casts above and what is excluded.
+
+### Included
+
+Items that are included in the basic costs of hosting the CHT:
+
+* Servers - This is the most likely place where Medic can leverage their intimate knowledge from hosting many dozen production instances of the CHT.  This includes:
+    * Average monthly cost to host the CHT
+    * Ability to adjust based on expected user count, number of workflows and a few other program specific numbers like how many workflows there are.
+    * Amount of CPU/RAM/Storage needed - while it is assumed this will be for AWS, it will come in the form of something like "20 vCPUs" which can be translated to any hosting environment.
+* Built in overprovisioning to allow for bursts and a bit of growth
+* Monitoring and Alerting - Since all software fails eventually, you need to be prepared to defend against this with aggressive monitoring and alerting.  The goal will be to fix the problem before any users notice.
+* Ingress/Egress - This is the ability to send and receive data to the CHT.  Pricing from Datacenters and Cloud is readily available and "Servers" above can not exist without it.
+* CHT Core updates and system maintenance - Both the CHT and underlying operating system will be upgraded on a regular basis.  This not only ensures there are no security vulnerabilities, but also ensures the deployments gains the benefit of new features and performance gains in the CHT.
+
+### Excluded
+
+There are many assumptions about what else is needed to run the CHT. While important, they are excluded:
+
+* Kubernetes cluster - Current CHT production best practices include a Kubernetes cluster that will have a fixed monthly cost to operate, including the worker nodes which may be large in size (RAM & CPU).
+* Backups - Regular snapshots of the production data will need to be taken to ensure there is no data loss in case of catastrophic failure of server hardware.  This takes up a disk space which should be accounted for when budgeting to host the CHT.
```

Comment:
This feels like one we should be able to include without too much effort - was the data no available?